### ISSUE TITLE
Cannot change recurring location

### ISSUE THREAD

**Issue by cassidyjames** [GetTogetherComm/GetTogether#140]
I cannot seem to find a way to update the default location for a recurring event; for DOSR, we have a weekly event that rotates locations, but every week the event is created with the first location we ever used. 

First in series: https://gettogether.community/events/271/thursday-co-working/

Upcoming event (manually edited location after its creation): https://gettogether.community/events/508/thursday-co-working/

**Comment by mhall119** [comment]
If you know that you will always alternate between two locations, you can create two series (one for each) that happen every other week instead of every week.

**Comment by cassidyjames** [comment]
@mhall119 oh, I didn't realize there was a place to view and manage the series! It also looks like there's not a way to just remove a location from a series as far as I can find. So maybe those are the improvements:

1. Make it more obvious to get to/manage the series (i.e. when editing an event created from a series, maybe ask if the user wants to edit this single event or edit this and future events, or something).

2. Make it possible (or more obvious, if it's already possible) to remove a location from a series.

In our case, the location changes every week and the next location is determined at the end of the meetup. So I think what I'll need to do is create a new series with no location, and then just add the location to the automatically created events once they're created.

**Comment by cassidyjames** [comment]
I've create a new location-less series at https://gettogether.community/series/54/thursday-co-working/ and will see how that works for us.

**Comment by TheTincho** [comment]
I'd just add that the UI to find the event series is not super obvious, I did not see it until I had found this bug

**Comment by cassidyjames** [comment]
@mhall119 to add/follow-up on my previous comment: it appears that setting a location on any event in a series _also_ applies that location the the series. The result is that we cannot use a series for our type of event, where there is a known repeating time but a variable location. Instead, we've had to revert to manually creating events every week.

**Comment by mhall119** [comment]
@cassidyjames yes, if the Series has no Place defined, and you set a Place for an Event in that Series, it will assume you want that Place for all future Events in the Series. This was to support the use case of somebody creating a new recurring event before they knew what Place it would occur at.

I think it's beginning to make sense to include a confirmation page when modifying an event in a series, asking if the change should be made to just that event, or future events in the series also. This is how Google calendar does it anyway, do you agree with that approach?

**Comment by mhall119** [comment]
@TheTincho I've added a listing of recurring event series on the team's "Events" page now.

**Comment by cassidyjames** [comment]
@mhall119 that would be perfect from my view!
